docs/interactionmanager.md

```diff
@@ -37,6 +37,157 @@ By default, queued tasks are executed together in a loop in one `setImmediate` b
 
 ---
 
+## Example
+
+### Basic
+
+```SnackPlayer name=InteractionManager%20Function%20Component%20Basic%20Example&supportedPlatforms=ios,android
+import React, { useEffect } from "react";
+import {
+  Alert,
+  Animated,
+  InteractionManager,
+  Platform,
+  StyleSheet,
+  Text,
+  View
+} from "react-native";
+
+const instructions = Platform.select({
+  ios: "Press Cmd+R to reload,\n" + "Cmd+D or shake for dev menu",
+  android:
+    "Double tap R on your keyboard to reload,\n" +
+    "Shake or press menu button for dev menu"
+});
+
+const useMount = func => useEffect(() => func(), []);
+
+const useFadeIn = (duration = 5000) => {
+  const [opacity] = React.useState(new Animated.Value(0));
```

Comment:
`useState` should be also imported at the beginning of the file, no need to use static call here.